### DISCUSSION TITLE
Local Business Store Schema Type

### DISCUSSION THREAD

**Discussion by Mellyod** [schemaorg/schemaorg#2979] [Q&A]
What should be selected for a "Collectibles Store" schema type? Collectibles Stores have a wide variety of products in different product categories which are a combination of Hobby Store, Home Goods Store, Toy Store, etc. By just selecting one of these store types, it excludes the products categorized under a different type and Collectible Stores are smaller than Department Stores. Your assistance is appreciated.

**Comment by WeaverStever** [comment]
In the classic sense, DepartmentStore generally meant a large variety of products.

Whatever subtype you decide to use, in this example I added additionalType with the WikiData url for Collectible.
You might also look into https://schema.org/hasOfferCatalog to define your specialties.

~~~~
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "DepartmentStore",
  "additionalType": "https://www.wikidata.org/wiki/Q2342494",
  "address": {
    "@type": "PostalAddress",
    "addressLocality": "Mexico Beach",
    "addressRegion": "FL",
    "streetAddress": "3102 Highway 98"
  },
  "description": "A superb collection of fine gifts and clothing to accent your stay in Mexico Beach.",
  "name": "Beachwalk Beachwear & Giftware",
  "telephone": "850-648-4200"
}
</script>


~~~~

**Comment by gmboxsmg2psn-collab** [comment]
Las [tragamonedas populares](https://orobet.do/es/casino/popular-games) suelen llamar la atención por sus temas visuales, mecánicas sencillas y rondas especiales. En OroBet, quienes disfrutan de este tipo de entretenimiento pueden explorar distintas categorías de juegos y elegir la opción que mejor encaje con sus gustos, siempre con un enfoque de juego responsable.
